lib/linguist/languages.yml

```diff
@@ -8116,10 +8116,16 @@ Zimpl:
   extensions:
   - ".zimpl"
   - ".zmpl"
-  - ".zpl"
   tm_scope: none
   ace_mode: text
   language_id: 411
+ZPL:
+  type: programming
+  color: "#4A4A4A"
+  extensions:
+  - ".zpl"
+  tm_scope: none
+  ace_mode: text
```

Comment:
You're not finished following the documentation for adding a language 😉 Please do so.